docs/getting-started/accessing-your-fleet.md

```diff
@@ -0,0 +1,45 @@
+# Accessing Your Fleet
+
+In order to complete Fleet operations, you need to access the **aux node**. From here you can run the `fleet` commands.
+
+## Log in to the aux node
+
+Test you can log into the **aux node**.
+
+```bash
+ssh -t deploy@aux.{fleet-id}.f.nchr.io
+```
+
+An easier way to access fleet is to set up an alias.
+Future examples will assume you have aliased the `fleet` command
+
+```bash
+alias fleet='ssh -t -o LogLevel=QUIET deploy@aux.{fleet-id}.f.nchr.io --'
+```
+
+Once you're done with the test and alias setup above, you should be able to type the following in your terminal and get the list of available commands
```

Comment:
"type the following in your terminal"

Should read

"type the following in to your terminal"